docs/concepts.rst

```diff
@@ -5,13 +5,13 @@ Django's traditional view of the world revolves around requests and responses;
 a request comes in, Django is fired up to serve it, generates a response to
 send, and then Django goes away and waits for the next request.
 
-That was fine when the internet was all driven by simple browser interactions,
+That was fine when the internet was driven by simple browser interactions,
 but the modern Web includes things like WebSockets and HTTP2 server push,
 which allow websites to communicate outside of this traditional cycle.
 
 And, beyond that, there are plenty of non-critical tasks that applications
 could easily offload until after a response as been sent - like saving things
-into a cache, or thumbnailing newly-uploaded images.
+into a cache, sending emails or thumbnailing newly-uploaded images.
```

Comment:
That makes sense. I'll remove it to avoid scope creep for the PR. I do think it would be helpful to have a list of common task, and mentioning how well they could be achieved with channels.